beacon-chain/state/phase0.go

```diff
@@ -148,6 +148,7 @@ type WriteOnlyBlockRoots interface {
 
 // WriteOnlyStateRoots defines a struct which only has write access to state roots methods.
 type WriteOnlyStateRoots interface {
+	SetStateRoots(val *[8192][32]byte) error
```

Comment:
I brought this method back because it is used in `beacon-chain/core/altair/upgrade.go`,